gbif-occurrence-linkage/main.py

```diff
@@ -2,102 +2,70 @@
 import logging
 import os
 import uuid
-from datetime import datetime, timezone
 from typing import Dict
 
 import requests
 from kafka import KafkaConsumer, KafkaProducer
+import shared
 
 logging.basicConfig(format='%(asctime)s - %(message)s', level=logging.INFO)
 
-ODS_TYPE = "ods:type"
-ODS_ID = "ods:id"
-
-
 def start_kafka() -> None:
     """
     Start a kafka listener and process the messages by unpacking the image.
     When done it will republish the object, so it can be validated and storage by the processing service
     """
     consumer = KafkaConsumer(os.environ.get('KAFKA_CONSUMER_TOPIC'),
                              group_id=os.environ.get('KAFKA_CONSUMER_GROUP'),
-                             bootstrap_servers=[os.environ.get('KAFKA_CONSUMER_HOST')],
-                             value_deserializer=lambda m: json.loads(m.decode('utf-8')),
+                             bootstrap_servers=[
+                                 os.environ.get('KAFKA_CONSUMER_HOST')],
+                             value_deserializer=lambda m: json.loads(
+                                 m.decode('utf-8')),
                              enable_auto_commit=True)
-    producer = KafkaProducer(bootstrap_servers=[os.environ.get('KAFKA_PRODUCER_HOST')],
-                             value_serializer=lambda m: json.dumps(m).encode('utf-8'))
+    producer = KafkaProducer(
+        bootstrap_servers=[os.environ.get('KAFKA_PRODUCER_HOST')],
+        value_serializer=lambda m: json.dumps(m).encode('utf-8'))
     for msg in consumer:
         try:
             logging.info('Received message: ' + str(msg.value))
             json_value = msg.value
-            specimen_data = json_value['object']['digitalSpecimen']
+            shared.mark_job_as_running(json_value["jobId"])
```

Comment:
Is it jobId or jobID? (And dot notation)

Reply:
jobId isn't an opends term, it's called jobId, see here: https://github.com/DiSSCo/openDS/blob/master/data-model/developer-schema/annotation/0.3.0/schema/annotation-processing-event.json

Reply:
Ok not to complain but maybe its best to align this with all the other IDs in the schemas, even if its not ods.
Something to discuss with @samleeflang also